packages/salesforcedx-vscode-core/src/commands/auth/orgLoginWeb.ts

```diff
@@ -208,31 +206,31 @@ export abstract class AuthDemoModeExecutor<
 
 export class OrgLoginWebDemoModeExecutor extends AuthDemoModeExecutor<AuthParams> {
   public build(data: AuthParams): Command {
-    return new SfdxCommandBuilder()
+    return new SfCommandBuilder()
       .withDescription(nls.localize('org_login_web_authorize_org_text'))
       .withArg(CLI.ORG_LOGIN_WEB)
       .withFlag('--alias', data.alias)
       .withFlag('--instance-url', data.loginUrl)
       .withArg('--set-default')
       .withArg('--no-prompt')
       .withJson()
-      .withLogName('force_auth_web_login_demo_mode')
+      .withLogName('org_login_web_demo_mode')
```

Comment:
Does this command no longer exist in SF v2?  I found this article that says demo mode is now set using an environment variable: https://developer.salesforce.com/tools/vscode/en/user-guide/demo-mode

Reply:
It has always used that env var. You can find the config in /salesforcedx-vscode/packages/salesforcedx-vscode-core/src/modes/demo-mode.ts